src/rules/no-identical-functions.ts

```diff
@@ -32,7 +34,18 @@ type FunctionNode =
 const message =
   'Update this function so that its implementation is not identical to the one on line {{line}}.';
 
-const rule: TSESLint.RuleModule<string, string[]> = {
+type Options = (number | 'sonar-runtime')[];
+
+const OPTION_SCHEMA = {
+  anyOf: [
+    { type: 'integer', minimum: 3 },
+    {
+      enum: ['sonar-runtime'],
+    },
+  ],
```

Comment:
I am not sure, but we need to test locally if this `anyOf` property will still work with the following logic in SonarJS:

https://github.com/SonarSource/SonarJS/blob/55324ff203c52d325c49b45434e6112551d59fb1/eslint-bridge/src/linter.ts#L332-L357

Reply:
definitely it will not:) good catch. I will see what I can do

Reply:
👍